pkg/networkmanager/index.html

```diff
@@ -104,27 +104,30 @@ <h1 id="testing-connection-curtain-restoring" translate="yes">Restoring connecti
         {{! member interfaces will be rendered into here as a list of checkboxes }}
       </div>
 
-      <label class="control-label" for="network-bridge-settings-stp-enabled-input" translate="yes">Spanning tree protocol (STP)</label>
+      <label class="control-label" for="network-bridge-settings-stp-enabled-input" translate="yes">Options</label>
       <label class="checkbox-inline">
         <input id="network-bridge-settings-stp-enabled-input" type="checkbox"
           {{#stp_checked}}checked{{/stp_checked}}/>
+        <span translate="yes">Spanning tree protocol (STP)</span>
```

Comment:
This STP checkbox is about enabling / disabling STP, and then it exposes some options. Therefore I think this needs different handling

How about labeling the checkbox that enables STP and exposes these options like: `Use spanning tree mode` and then have the Options bellow?

Otherwise this can also use a Switch instead of a checkbox.

Reply:
In other words - I don't really like nesting the main checkbox for STP under generic Options label. It feels off there especially because all options then are related to STP.

Reply:
> I don't really like nesting the main checkbox for STP under generic Options label

You're right. Options is too generic here, especially as we only have one. STP is a topology option, so I'm changing the label to something more specific: Topology, which is what STP manages.

There's also the newer OSPF (Open Shortest Path First) and EIGRP (Enhanced Interior Gateway Routing Protocol) in the same family of solutions to provide alternate paths, but we only support STP.

---

> Otherwise this can also use a Switch instead of a checkbox.

Nope, this shouldn't be a switch. It doesn't direct apply, as it needs a save state. Therefore, it's a checkbox, not a switch.

From https://www.patternfly.org/v4/components/switch/design-guidelines/#when-to-use-switches-versus-checkboxes:
> A switch changes an option and saves it simultaneously, while checkboxes require a separate action to save the selection, such as pressing a “Submit” or “Save” button.